### PR TITLE
Create models - controllers

### PR DESCRIPTION
**Pull request by tanguydamois**
Hi guys,
I created the controllers - views & models
Did all the models
created controllers for users and tennis-courts
worked on the view of #index
I also add the routes for the tennis_courts (CRUD), I don t know why I am not able to see the /tennis-courts page after created the "def index" that s why i commented it in the controller. It should be something very easy but I can see it.

### CODE REVIEW

**Comment by lcskoerner** [comment]
app/views/pages/index.html.erb should be placed on app/views/tennis_courts/ commit cbe431b

**Comment by lcskoerner** [comment]
Review model should be made in a different branch 43f723c

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Rails.application.routes.draw do
-  root to: 'pages#home'
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root to: 'pages#index'
```
Comment (lcskoerner) [inline config/routes.rb:2]: root to: 'tennis_courts#index'

**Comment by tanguydamois** [comment]
> Did you generate the users using devise? commit [bf320af](https://github.com/lcskoerner/tennis-where/commit/bf320af1202c340de5bf50f88d36af97a7d829b9)

My bad i did not. I tried to remove the table and create the model again using devise but it didn't work. i am sure you will find a way... I removed the controller and the view but the model is style here.

File: config/routes.rb (hunk 1 above)
Comment (tanguydamois) [reply config/routes.rb:2]: I made the correction and will push it

**Comment by tanguydamois** [comment]
> app/views/pages/index.html.erb should be placed on app/views/tennis_courts/ commit [cbe431b](https://github.com/lcskoerner/tennis-where/commit/cbe431b97443f9880927045a88526769776dd53b)
> this is why the view is not being loaded.

i am confuses it looks like it is already in app/views/tennis_courts/index.rb

**Comment by tanguydamois** [comment]
> Review model should be made in a different branch [43f723c](https://github.com/lcskoerner/tennis-where/commit/43f723ca70bd840f74b1eefb7e79adce8c88044c)
> Booking model should be made in a different branch [0dd28ab](https://github.com/lcskoerner/tennis-where/commit/0dd28abaea9f3c85bc65ab6f3d46f65830f9bc9a)

ok i will split it in the futur

**Pull request merged**